components/scream/scripts/utils.py

```diff
@@ -351,3 +351,47 @@ def get_common_ancestor(other_head, head="HEAD", repo=None):
     """
     rc, output, _ = run_cmd("git merge-base {} {}".format(other_head, head), from_dir=repo)
     return output if rc == 0 else None
+
+###############################################################################
+def update_submodules(repo=None):
+###############################################################################
+    """
+    Updates submodules
+    """
+    run_cmd("git submodule update --init --recursive",from_dir=repo)
```

Comment:
I think this should be run_cmd_no_fail; we don't want to tolerate failures to update submodules.

Reply:
True.